jetstream/cli.py

```diff
@@ -522,6 +551,47 @@ def rerun_config_changed(
         sys.exit(0 if success else 1)
 
 
+@cli.command("ensure_enrollments")
+@project_id_option
+@dataset_id_option
+@click.option(
+    "--date",
+    type=ClickDate(),
+    help="Date for which experiments should be analyzed",
+    metavar="YYYY-MM-DD",
+    required=True,
+)
```

Comment:
Could we just always use the current date? The output doesn't depend on this except for whether we're ready to create the table yet or not.